### DISCUSSION TITLE
Do the following two ways of providing data have any effect on the training results？

### DISCUSSION THREAD

**Discussion by ZKNB** [deepmodeling/deepmd-kit#3131] [Q&A]
1. There's an OUTCAR file for 200 fps AIMD.
2. The position information of each frame system in the OUTCAR file in 1 is extracted to do the single-point energy calculation, and 200 OUTCAR static calculation files are obtained.

**Comment by njzjz** [comment]
They should give the same results if you can ensure other configurations are the same.
